test/spec/modules/adxcgBidAdapter_spec.js

```diff
@@ -14,19 +14,57 @@ describe('AdxcgAdapter', () => {
       'bidId': '84ab500420319d',
       'bidderRequestId': '7101db09af0db2',
       'auctionId': '1d1a030790a475',
-    };
+    }
+
+    let bidVideo = {
+      'bidder': 'adxcg',
+      'params': {
+        'adzoneid': '1',
+        'api': [2],
+        'protocols': [1, 2],
+        'mimes': ['video/mp4', 'video/x-flv'],
+        'maxduration': 30
+      },
+      'mediaTypes': {
+        'video': {
+          'context': 'instream'
+        }
+      },
+      'adUnitCode': 'adunit-code',
+      'sizes': [[300, 250], [640, 360], [1, 1]],
+      'bidId': '84ab500420319d',
+      'bidderRequestId': '7101db09af0db2',
+      'auctionId': '1d1a030790a475',
+    }
 
     it('should return true when required params found', () => {
-      expect(spec.isBidRequestValid(bid)).to.equal(true);
-    });
+      expect(spec.isBidRequestValid(bidBanner)).to.equal(true)
+    })
+
+    it('should return true when required params not found', () => {
+      expect(spec.isBidRequestValid({})).to.be.false
+    })
 
     it('should return false when required params are not passed', () => {
-      let bid = Object.assign({}, bid);
-      delete bid.params;
-      bid.params = {};
-      expect(spec.isBidRequestValid(bid)).to.equal(false);
-    });
-  });
+      let bid = Object.assign({}, bidBanner)
+      delete bid.params
+      bid.params = {}
+      expect(spec.isBidRequestValid(bid)).to.equal(false)
+    })
+
+    it('should return true when required video params not found', () => {
+      const simpleVideo = JSON.parse(JSON.stringify(bidVideo))
+      simpleVideo.params.adzoneid = 123
+      expect(spec.isBidRequestValid(simpleVideo)).to.be.false
+      // simpleVideo.params.site_id = siteId;
```

Comment:
Remove commented code here, line 64 and line 85

Reply:
done

Reply:
Can you also delete line 59